test/views/templates/ChangeSuccessViewSpec.scala

```diff
@@ -43,110 +44,114 @@ class ChangeSuccessViewSpec extends ViewBaseSpec {
+        val viewModel = ChangeSuccessViewModel(exampleTitle, None, None, Some(ContactPreference.digital))
+        lazy val view = injectedView(viewModel)(messages, mockConfig, User("1111111111"))
 
         lazy implicit val document: Document = Jsoup.parse(view.body)
 
         "have the page title provided by the model" in {
-          elementText(Selectors.title) shouldBe s"$exampleTitle - Your client’s VAT details - GOV.UK"
+          elementText(Selectors.title) shouldBe s"$exampleTitle - Business tax account - GOV.UK"
         }
 
         "have the heading provided by the model" in {
           elementText(Selectors.pageHeading) shouldBe exampleTitle
         }
 
+        "have the correct second heading" in {
+          elementText(Selectors.secondaryHeading) shouldBe "What happens next"
+        }
+
         "have a finish button with the correct text" in {
-          elementText(Selectors.button) shouldBe "Finish"
+          elementText(Selectors.button) shouldBe "View your business details"
         }
 
         "have a finish button which navigates to the Change of Circs overview page" in {
           element(Selectors.button).select("a").attr("href") shouldBe mockConfig.manageVatSubscriptionServicePath
         }
 
         "have the correct first paragraph" in {
-          elementText(Selectors.paragraphOne) shouldBe "We’ll send an email to agent@example.com within 2" +
-            " working days telling you whether or not the request has been accepted."
+          elementText(Selectors.paragraphOne) shouldBe "We’ll send you an email within 2 working days, telling you whether we can accept the trading name."
+        }
+
+        "have the correct second paragraph" in {
+          elementText(Selectors.paragraphTwo) shouldBe "You can also check your secure messages for an update."
         }
       }
 
-      "the agent doesn't have an email address registered" should {
-        val viewModel = ChangeSuccessViewModel(exampleTitle, None, Some("TheBusiness"))
-        lazy val view = injectedView(viewModel)(messages, mockConfig, User("1111111111", arn = Some(BaseTestConstants.arn)))
+      "the contact preference is paper" when {
+        val viewModel = ChangeSuccessViewModel(exampleTitle, None, None, Some(ContactPreference.paper))
+        lazy val view = injectedView(viewModel)(messages, mockConfig, User("1111111111"))
 
         lazy implicit val document: Document = Jsoup.parse(view.body)
 
-        "have the page title provided by the model" in {
-          elementText(Selectors.title) shouldBe s"$exampleTitle - Your client’s VAT details - GOV.UK"
+        "have the correct first paragraph" in {
+          elementText(Selectors.paragraphOne) shouldBe "We’ll send a letter to your principal place of business " +
+            "within 15 working days, telling you whether we can accept the trading name."
         }
+      }
```

Comment:
Could you add an additional test under here for a user whose contact preference is `None` (i.e. unable to retrieve it), and just check the paragraph text